File: beecrowd/2482.cpp
#include <iostream>
#include <string>
#include <map>

using namespace std;

int main()
{
    map<string, string> dict = {};

    int dict_length;

    cin >> dict_length;

    for (int i = 0; i < dict_length; i++)
    {
        string idioma, palavra;

        cin >> idioma;

        std::getline(std::cin >> std::ws, palavra);

        pair<string, string> par = {idioma, palavra};

        dict.insert(par);
    }

    int child;

    cin >> child;

    for (int i = 0; i < child; i++)
    {
        string nome;
        string idioma;

        std::getline(std::cin >> std::ws, nome);
        cin >> idioma;

        cout << nome << "\n";
        cout << dict[idioma] << "\n\n";
    }
    return 0;
}

File: beecrowd/2482.py
# -*- coding: utf-8 -*-

traducao = {}

n = int(input())

for i in range(n):
    idioma = input()
    palavra = input()
    traducao[idioma] = palavra

m = int(input())

for i in range(m):
    nome = input()
    idioma = input()
    print(nome)
    print(traducao[idioma])
    print()
